src/sd.jl

```diff
@@ -25,3 +25,11 @@ function addconstraint(m::Model, c::SDVariableConstraint)
     cref = MOI.addconstraint!(m.instance, MOI.VectorOfVariables([instancereference(c.Q[i, j]) for j in 1:n for i in 1:j]), MOI.PositiveSemidefiniteConeTriangle(n))
     return ConstraintRef(m, cref)
 end
+
+function constructconstraint!(x::AbstractMatrix, ::PSDCone)
+    n = Base.LinAlg.checksquare(x)
+    @assert issymmetric(x)
```

Comment:
Could you add a comment here discussing the plans for handling nonsymmetric matrices?